### DISCUSSION TITLE
How to embed images from url?

### DISCUSSION THREAD

**Discussion by winkelement** [outline/outline#4605] [General]
I am sorry if this is a stupid question but i think i am missing something obvious so maybe someone can point me in the right direction:
How can i embed an image that is hosted somewhere else? The only options i see are 1. uploading an image and 2. adding a link.
I just want to do something like i would in normal markdown like `![Title](https://link.to/image.jpg)`
Is this not possible?

**Comment by tommoor** [comment]
It would be nice to have some UI for this, but you can type out the markdown as you have above and it should work
